sdk/lib/crt/stdio/stat64.c

```diff
@@ -14,29 +14,28 @@ HANDLE fdtoh(int fd);
 #define TCHAR4 ULONG
 #endif
 
-#define TCSIZE sizeof(_TCHAR)
-#define TOULL(x) ((TCHAR4)(x))
+#define TCSIZE_BITS sizeof(_TCHAR)*8
```

Comment:
It's good practice to put parentheses around the macro expressions so you don't get surprised by operator precedence issues when using them

Reply:
Fixed